README.rst

```diff
@@ -103,10 +103,11 @@ Authors
 =======
 
 Ansible was created by `Michael DeHaan <https://github.com/mpdehaan>`_
-(michael.dehaan/gmail/com) and has contributions from over 1000 users
+(michael.dehaan/gmail/com) and has contributions from over 3700 users
 (and growing). Thanks everyone!
 
-Ansible is sponsored by `Ansible, Inc <https://ansible.com>`_
+`Ansible <https://www.ansible.com>`_ is sponsored by `Red Hat, Inc.
```

Comment:
@acozine notice the period in the link. Technically it's at the end of the line, but inside of a link. Should I move it outside? Otherwise, it'd be two dots in the end if one is inside the link and the other's outside.

Reply:
Ah, you're right, I missed that. Thanks, and merging.